cvat/apps/engine/serializers.py

```diff
@@ -134,66 +195,129 @@ class Meta:
         fields = ('id', 'svg',)
 
 class LabelSerializer(SublabelSerializer):
-    deleted = serializers.BooleanField(required=False, help_text='Delete label if value is true from proper Task/Project object')
+    deleted = serializers.BooleanField(required=False, write_only=True,
+        help_text='Delete the label. Only applicable in the PATCH methods of a project or a task.')
     sublabels = SublabelSerializer(many=True, required=False)
     svg = serializers.CharField(allow_blank=True, required=False)
+    has_parent = serializers.BooleanField(read_only=True, source='has_parent_label', required=False)
 
     class Meta:
         model = models.Label
-        fields = ('id', 'name', 'color', 'attributes', 'deleted', 'type', 'svg', 'sublabels', 'has_parent')
+        fields = (
+            'id', 'name', 'color', 'attributes', 'deleted', 'type', 'svg',
+            'sublabels', 'project_id', 'task_id', 'parent_id', 'has_parent'
```

Comment:
As we discussed need to keep only one field `has_parent` or `parent_id` in the next PR (probably parent_id more informative)